src/Voron/Data/Compression/DecompressionBuffersPool.cs

```diff
@@ -172,13 +167,13 @@ public IDisposable GetTemporaryPage(LowLevelTransaction tx, int pageSize, out Te
                     }
 
                     buffer = new DecompressionBuffer(_compressionPager, _lastUsedPage, pageSize, this, index, tx);
-
+                    _compressionPager.TryUse();
```

Comment:
Shouldn't we care about the result returning by this method?

Reply:
No, we are using the current pager here. the problem is with the old pagers